src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -225,6 +225,15 @@
   <data name="DoNotUseDeprecatedSecurityProtocolsMessage" xml:space="preserve">
     <value>Hard-coded use of deprecated security protocol {0}</value>
   </data>
+  <data name="DoNotUseDSA" xml:space="preserve">
+    <value>Do Not Use Digital Signature Algorithm (DSA)</value>
+  </data>
+  <data name="DoNotUseDSADescription" xml:space="preserve">
+    <value>DSA is too weak to use. Any creation or return of DSA will be flagged a diagnositc.</value>
+  </data>
+  <data name="DoNotUseDSAMessage" xml:space="preserve">
+    <value>Weak asymmetric encryption algorithm {0} is banned by the SDL. Switch to an RSA with at least 2048 key size, ECDH or ECDSA algorithm instead</value>
```

Comment:
>Weak asymmetric encryption algorithm {0} is banned by the SDL [](start = 11, length = 61)

Asymmetric encryption algorithm {0} is weak.

(probably shouldn't mention the SDL in public rules, and instead provide underlying justifications for the SDL)